Automattic/wp-calypso
# Staging Sites: custom my home for staging sites

Fixes https://github.com/Automattic/dotcom-forge/issues/1900

## Proposed Changes

Some My-home elements are not relevant for staging sites. This pr introduces a custom my-home for those sites.

## Testing Instructions

### Test new my home

1. Makes sure you have applied D105402-code to your sandbox.
2. Create a staging site
3. Visit `/home/<your site>`
4. Ensure that you see something like the following:

![home-card](https://user-images.githubusercontent.com/497103/227244195-5f8315d7-b05f-4bc6-97d6-e627d5ea24b5.png)

## Pre-merge Checklist
